src/components/data_submission/DataSubmissionNihAnvilUse.js

```diff
@@ -0,0 +1,87 @@
+import {div, h, h2} from 'react-hyperscript-helpers';
+import {useState} from 'react';
+import {FormField, FormFieldTypes, FormValidators} from '../forms/forms';
+
+
+export default function DataSubmissionNihAnvilUse(props) {
+  const { onChange } = props;
+  const [nihAnvilUse, setNihAnvilUse] = useState();
+
+  const I_DID = 'I did';
+  const I_WILL = 'I will';
+  const NO = 'No';
+
+  return h(div, {
+    style: {
+      padding: '50px 0',
+      maxWidth: 800,
+      margin: 'auto'
+    }
+  }, [
+    h2('NIH and AnVIL use'),
+    h(FormField, {
+      id: 'nihAnvilUse',
+      title: 'Will you or did you submit data to the NIH?',
+      type: FormFieldTypes.RADIO,
+      selectOptions: [
+        {label: I_DID, value: I_DID},
+        {label: I_WILL, value: I_WILL},
+        {label: NO, value: NO},
+      ],
+      dependentFormFields: [
+        {id: 'submittingToAnvil', type: FormFieldTypes.RADIO, shouldClear: ({value}) => value === I_DID},
```

Comment:
I originally had all of the dependent form fields be cleared when `nihAnvilUse` is changed, but there are two values of `nihAnvilUse`  where the `submittingToAnvil` question appears. So, if I switched between `nihAnvilUse === 'I will'` and `'nihAnvilUse' === 'No'`, `submittingToAnvil` in formData is cleared but the selected radio buttons weren't. One possible solution is allowing the `dependentFormFields` to have a `shouldClear` condition (if not provided, assume clear on any change). Here, `submittingToAnvil` is only cleared in the formData when the question is no longer rendered, so that the value in formData and the selected radio button are consistent with each other